jnosql-dynamodb/dynamoDockerRun.sh

```diff
@@ -1,8 +1,7 @@
 AWS_ACCESS_KEY_ID=$(aws --profile default configure get aws_access_key_id)
 AWS_SECRET_ACCESS_KEY=$(aws --profile default configure get aws_secret_access_key)
```

Comment:
@dearrudam can we remove this file?

Reply:
sure, @otaviojava!!! I didn't remove it 'cause I thought that it would be helpful during local development maybe...

Reply:
The local development should be using an integration test with a test container.
I guess it was a legacy move.